packages/playback-core/src/index.ts

```diff
@@ -263,7 +274,13 @@ export const loadMedia = (
   props: Partial<
     Pick<
       MuxMediaProps,
-      "preferMse" | "src" | "type" | "startTime" | "streamType" | "autoplay"
+      | "preferMse"
+      | "src"
+      | "type"
+      | "startTime"
+      | "streamType"
+      | "autoplay"
+      | "autoPlay"
```

Comment:
nit: I don't think we need `autoPlay` here.

Reply:
yeah, I was just thinking that these aren't necessary anymore. I'll try it out tomorrow.